tests/slurm_command_gen_strategy/test_sleep.py

```diff
@@ -0,0 +1,56 @@
+# SPDX-FileCopyrightText: NVIDIA CORPORATION & AFFILIATES
+# Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, List
+
+import pytest
+from cloudai.schema.test_template.sleep.slurm_command_gen_strategy import SleepSlurmCommandGenStrategy
+from cloudai.systems import SlurmSystem
+
+
+class TestSleepSlurmCommandGenStrategy:
+    def get_test_command(
+        self, slurm_system: SlurmSystem, env_vars: Dict[str, str], cmd_args: Dict[str, str], extra_cmd_args: str
+    ) -> List[str]:
+        return SleepSlurmCommandGenStrategy(slurm_system, {}).generate_test_command(env_vars, cmd_args, extra_cmd_args)
+
+    @pytest.mark.parametrize(
+        "env_vars, cmd_args, extra_cmd_args, expected_command",
```

Comment:
As you are not permutating `env_vars` and `extra_cmd_args`, IMO it would be better to remove the to simplify the test.